### ISSUE TITLE
Extending IF functionality

### ISSUE THREAD

**Issue by Krys-P** [SeleniumHQ/selenium-ide#786]
## 🚀 Feature Proposal

Extend IF functionality to include;
IF verify element not present = Failed/OK

## Motivation

If a login page appears, then login, if already logged in, then ignore step.

## Example

If header of page says "login" then enter username or password, if not, then next step.

**Comment by corevo** [comment]
Let me rephrase, allow to `if` all assertions, like `if | assert element preset`

**Comment by david-nowinski** [comment]
You can emulate this by using a store xpath count then checking if > 0. I used this trick quite a lot for conditional branching, especially testing login with oauth2 where it can get quite wild sometimes.
